sender.go

```diff
@@ -60,6 +60,7 @@ func (s *Sender) Send(ctx context.Context, msg *Message) error {
 // send is separated from Send so that the mutex unlock can be deferred without
 // locking the transfer confirmation that happens in Send.
 func (s *Sender) send(ctx context.Context, msg *Message) (chan deliveryState, error) {
+	const maxDeliveryTagLength = 32
```

Comment:
You're very thorough. Or a robot.

Jury is still out on that.

Reply:
Can't be both?  :p